src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -171,6 +171,22 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructorReminders(
     }
     
     /**
+     * Generates the feedback session closed emails for the given {@code session}.
+     */
+    public List<EmailWrapper> generateFeedbackSessionClosedEmails(FeedbackSessionAttributes session) {
+
+        boolean isEmailNeeded = fsLogic.isFeedbackSessionForStudentsToAnswer(session);
```

Comment:
Sorry, missed this one. Next time, ping me in the main conversation thread, not inside code. 
Now that I think about it, why not send the email to everyone? The only downside is the extra cost but I think that's ok.